x-pack/test/functional/apps/monitoring/elasticsearch/nodes_mb.js

```diff
@@ -189,19 +189,21 @@ export default function ({ getService, getPageObjects }) {
         });
       });
 
-      // this is actually broken, see https://github.com/elastic/kibana/issues/122338
-      it.skip('should sort by status', async () => {
-        const sortedStatusesAscending = ['Status: Offline', 'Status: Online', 'Status: Online'];
-        const sortedStatusesDescending = [...sortedStatusesAscending].reverse();
-
+      it('should sort by status', async () => {
         await nodesList.clickStatusCol();
-        await retry.try(async () => {
-          expect(await nodesList.getNodeStatuses()).to.eql(sortedStatusesDescending);
-        });
-
         await nodesList.clickStatusCol();
```

Comment:
Minor but I think it would be good to test before and after the sorting happens to see that it changes.

Reply:
This is meant to match the [nodes.mb](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/monitoring/elasticsearch/nodes.js#L194) file so I reverted felix's changes back.  We can update all the sort tests in another PR.